### ISSUE TITLE
Please add support for winget

### ISSUE THREAD

**Issue by Onekki** [zhongyang219/TrafficMonitor#718]

**Comment by golamrabbiazad** [comment]
Yes, It needs support for [winget](https://github.com/microsoft/winget-cli) package manager. Thanks!

**Comment by PlexPt** [comment]
+1

**Comment by ST-DDT** [comment]
This is somewhat supported

- `winget install zhongyang219.TrafficMonitor.Full`
- `winget install zhongyang219.TrafficMonitor.Lite`

but you might want to be careful with installing updates via winget:

- https://github.com/zhongyang219/TrafficMonitor/issues/2018
